plugin/entrySchema.go

```diff
@@ -31,6 +31,9 @@ func schema(e Entry) (*EntrySchema, error) {
 		if err != nil {
 			return nil, err
 		}
+		if graph == nil {
```

Comment:
What's this handling?

Reply:
Mocking out schema graphs in the tests. Needed to add that for the `walker` tests.